lightning/src/util/enforcing_trait_impls.rs

```diff
@@ -225,6 +234,8 @@ impl EnforcingSigner {
 pub struct EnforcementState {
 	/// The last counterparty commitment number we signed, backwards counting
 	pub last_counterparty_commitment: Option<u64>,
```

Comment:
Might it be worth documenting why this field's an `Option` and the rest are just u64's?

Reply:
actually, it can be a u64 - changed